.github/workflows/jest.yml

```diff
@@ -0,0 +1,55 @@
+name: Jest Tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'eask'
+      - '**.yml'
+      - lisp/**
+      - cmds/**
+      - src/**
+      - test/**
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
```

Comment:
Let's add `- '**/docs.yml'` here and ignore `doc` changes.

Reply:
I'm planning on removing `jest.yml` and restoring the original workflow files, since different test groups run on different os/version combinations. So, for example, `analyze.yml` will run `npm test-docker test/jest/analyze.test.js` instead of the `make` command.

Reply:
Ah, got it!